lib/checkstl.h

```diff
@@ -234,6 +237,8 @@ class CPPCHECKLIB CheckStl : public Check {
 
     void knownEmptyContainerError(const Token *tok, const std::string& algo);
 
+    void eraseIteratorOutOfBoundsError(const Token* ftok, const Token* itertok, const ValueFlow::Value* val = nullptr);
```

Comment:
I don't see that this is called in `getErrorMessages`. And no info is added in `classInfo`.